lib/ossert/stats/base.rb

```diff
@@ -74,14 +220,26 @@ def <<(other_stats)
         self
       end
 
+      # Create instance of the class Base
+      #
+      # Also initiates all instance variables listed in "/config/stats"
+      # for a given section of metrics.
+      #
+      # @return [Base] instance of the class
       def initialize
         self.class.attributes.each do |var, type|
           instance_variable_set("@#{var}", Kernel.const_get(type).new) if type
         end
       end
 
+      # Get a median value of elements
+      #
+      # @param values [Array<Numeric>] numeric values to search through
+      # @param default_value [Numeric] default result for empty values
+      #
+      # @return [Numeric] Median value for the given values
       def median(values, default_value: 0)
-        values = Array(values).sort
+        values = Array(values.compact).sort
```

Comment:
Somehow I got there empty values, so I had to use `#compact`. Need to investigate it deeper, because now it seems pretty illogical for me.